chanfitness/rate_limit.go

```diff
@@ -0,0 +1,40 @@
+package chanfitness
+
+import "time"
+
+var (
+	// rateLimitScale is the number of events we allow per rate limited
+	// tier. Increasing this value makes our rate limiting more lenient,
+	// decreasing it makes us less lenient.
+	rateLimitScale = 200
```

Comment:
Yeah, always difficult to hand wave a number for things like this. One thing I wanted to make sure of is that we don't punish peers that are restarting a "regular" number of times, because then we start getting worse uptime for legit peers. 

Did some rough calculations [here ](https://github.com/lightningnetwork/lnd/issues/4165#issuecomment-643252037) re what our worst case is for these values, but we can really plug anything in here, so open to suggestions!

Reply:
Maybe let's see what others think, I'm fine with it